pcsx2/x86/microVU_IR.h

```diff
@@ -330,6 +330,16 @@ class microRegAlloc
 		}
 	}
 
+	bool checkVFClamp(int regId)
+	{
+		if(CHECK_SKIPMPEGHACK)
```

Comment:
Just curious, why skipmpeg?

Reply:
I just wanted to pick a gamefix to hook it up to which was unlikely to affect 3d performance. No other reason :)